templates/basic-template/generate-types/Cargo.toml

```
@@ -0,0 +1,11 @@
+[package]
+name = "generate-types"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+program = { package= "{{project-name}}", path = "../program", default-features = false, features=['std'] }
+schemars = "0.8.16"
+serde_json = { version = "1.0", default-features = false}
```

```suggestion
serde_json = { version = "1.0" }
```

Since we're not compiling to Wasm here this is probably fine